### ISSUE TITLE
the bot stop doing "bot stuff" after working 1h

### ISSUE THREAD

**Issue by crusardri** [PokemonGoF/PokemonGo-Bot#676]
### Expected Behavior

Working full time

### Actual Behavior

the bot goes on strike after one hour of work.

### Steps to Reproduce

start the bot, wait 1h

### Other Information
OS: Win10
Git Commit: (run 'git log -n 1 --pretty=format:"%H"' and paste it here)
Last dev (I still do not know yet how to use git command T.T)

bot start at:
[2016-07-24 14:49:47] [x] PokemonGO Bot v1.0

bot stop work at:

[2016-07-24 15:53:37] [#] Now at Pokestop: Detras De La Monta+¦a - Spinning...
[2016-07-24 15:53:39] [+] Loot:
[2016-07-24 15:53:39] [+] 50 xp
[2016-07-24 15:53:39] [+] 3x Pokeball (Total: 207)
[2016-07-24 15:53:39] [+] 1x Super Potion (Total: 1)
[2016-07-24 15:53:39] [+] Recycling 1x Super Potion...
[2016-07-24 15:53:39] [+] Recycling success
[2016-07-24 15:53:39] [#] PokeStop on cooldown. Time left: 5.00 minutes
[2016-07-24 15:53:41] [#] Found fort 9ec4549da9ad486fac723849c2091b29.16 at distance 0.44km
[2016-07-24 15:53:41] [#] Need to move closer to Pokestop
[2016-07-24 15:53:41] [#] Walking from (40.421252, -3.680628) to (40.4155, -3.687753) for approx. 30.00 seconds
[2016-07-24 15:54:26] [#] Finished walking
[2016-07-24 15:54:26] [#] Arrived at Pokestop
[2016-07-24 15:54:28] [#] Now at Pokestop: Fuente De La Rosa - Spinning...
[2016-07-24 15:54:29] [+] Loot:
[2016-07-24 15:54:29] [+] 50 xp
[2016-07-24 15:54:30] [+] 2x Pokeball (Total: 209)
[2016-07-24 15:54:30] [+] 1x Greatball (Total: 32)
[2016-07-24 15:54:30] [+] 1x Ultraball (Total: 10)
[2016-07-24 15:54:30] [#] PokeStop on cooldown. Time left: 4.99 minutes
[2016-07-24 15:54:32] [#] Found fort 6e83d133b2854005a2df52bda0982f84.16 at distance 0.45km
[2016-07-24 15:54:32] [#] Need to move closer to Pokestop
[2016-07-24 15:54:32] [#] Walking from (40.4155, -3.687753) to (40.418743, -3.688033) for approx. 13.00 seconds
[2016-07-24 15:54:49] [#] Finished walking
[2016-07-24 15:54:49] [#] Arrived at Pokestop
[2016-07-24 15:54:51] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 15:55:01] [#] Found fort 47d698f79bd441e49033ceab64e4d6ce.16 at distance 0.47km
[2016-07-24 15:55:01] [#] Need to move closer to Pokestop
[2016-07-24 15:55:01] [#] Walking from (40.418743, -3.688033) to (40.419281, -3.68805) for approx. 2.00 seconds
[2016-07-24 15:55:02] [#] Finished walking
[2016-07-24 15:55:02] [#] Arrived at Pokestop
[2016-07-24 15:55:05] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 15:55:19] [#] Found fort 14c1a85610c445dfa9e8bd1cd237cfd7.12 at distance 0.68km
[2016-07-24 15:55:19] [#] Need to move closer to Pokestop
[2016-07-24 15:55:19] [#] Walking from (40.419281, -3.68805) to (40.4152, -3.690623) for approx. 17.00 seconds
[2016-07-24 15:55:49] [#] Finished walking
[2016-07-24 15:55:49] [#] Arrived at Pokestop

**Comment by fsimchen** [comment]
Same issue:
[2016-07-24 11:05:17] [#] Now at Pokestop: Giant Thumb's Up - Spinning...
[2016-07-24 11:05:20] [+] Loot:
[2016-07-24 11:05:20] [+] 100 xp
[2016-07-24 11:05:22] [+] 2x Pokeball (Total: 155)
[2016-07-24 11:05:23] [+] 1x Razz Berry (Total: 87)
[2016-07-24 11:05:24] [+] 3x Potion (Total: 4)
[2016-07-24 11:05:24] [+] Recycling 3x Potion...
[2016-07-24 11:05:25] [+] Recycling success
[2016-07-24 11:05:25] [#] PokeStop on cooldown. Time left: 4.89 minutes
[2016-07-24 11:05:27] [#] Found fort c7ad03bd382c4aa1a4faab0bcdc656c0.16 at distance 0.94km
[2016-07-24 11:05:27] [#] Need to move closer to Pokestop
[2016-07-24 11:05:27] [#] Walking from (40.774282, -73.962301) to (40.776851, -73.977765) for approx. 54.00 seconds
[2016-07-24 11:08:26] [#] Finished walking
[2016-07-24 11:08:27] [#] Arrived at Pokestop
[2016-07-24 11:08:29] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:08:41] [#] Found fort 4ca3b1fe6bc44fafa5b88c9d0323fb7d.16 at distance 0.96km
[2016-07-24 11:08:41] [#] Need to move closer to Pokestop
[2016-07-24 11:08:41] [#] Walking from (40.776851, -73.977765) to (40.778112, -73.978101) for approx. 6.00 seconds
[2016-07-24 11:08:54] [#] Finished walking
[2016-07-24 11:08:55] [#] Arrived at Pokestop
[2016-07-24 11:08:57] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:09:12] [#] Found fort 84250c64643d4911b24bc6575e485ad0.16 at distance 0.97km
[2016-07-24 11:09:12] [#] Need to move closer to Pokestop
[2016-07-24 11:09:12] [#] Walking from (40.778112, -73.978101) to (40.778763, -73.978103) for approx. 3.00 seconds
[2016-07-24 11:09:21] [#] Finished walking
[2016-07-24 11:09:22] [#] Arrived at Pokestop
[2016-07-24 11:09:26] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:09:40] [#] Found fort cce3814a712a4f598d59308ba12d9cc6.16 at distance 0.98km
[2016-07-24 11:09:40] [#] Need to move closer to Pokestop
[2016-07-24 11:09:40] [#] Walking from (40.778763, -73.978103) to (40.777328, -73.978339) for approx. 7.00 seconds
[2016-07-24 11:09:56] [#] Finished walking
[2016-07-24 11:09:57] [#] Arrived at Pokestop
[2016-07-24 11:10:01] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:10:15] [#] Found fort b7952485401f49e197315f812afcea69.16 at distance 1.03km
[2016-07-24 11:10:15] [#] Need to move closer to Pokestop
[2016-07-24 11:10:15] [#] Walking from (40.777328, -73.978339) to (40.777586, -73.978858) for approx. 3.00 seconds
[2016-07-24 11:10:22] [#] Finished walking
[2016-07-24 11:10:23] [#] Arrived at Pokestop
[2016-07-24 11:10:27] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:10:38] [#] Found fort 88ce3660a85a4c4aa18c3518f69fa55f.16 at distance 1.03km
[2016-07-24 11:10:38] [#] Need to move closer to Pokestop
[2016-07-24 11:10:38] [#] Walking from (40.777586, -73.978858) to (40.778606, -73.978837) for approx. 5.00 seconds
[2016-07-24 11:10:51] [#] Finished walking
[2016-07-24 11:10:52] [#] Arrived at Pokestop
[2016-07-24 11:10:54] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:11:02] [#] Found fort 1656e112eec14cdba099674ffb4b064b.16 at distance 1.04km
[2016-07-24 11:11:02] [#] Need to move closer to Pokestop
[2016-07-24 11:11:02] [#] Walking from (40.778606, -73.978837) to (40.778253, -73.97905) for approx. 2.00 seconds
[2016-07-24 11:11:06] [#] Finished walking
[2016-07-24 11:11:07] [#] Arrived at Pokestop
[2016-07-24 11:11:11] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:11:24] [#] Found fort 9a4f369ded874dd5b11e42006cb50819.11 at distance 1.08km
[2016-07-24 11:11:24] [#] Need to move closer to Pokestop
[2016-07-24 11:11:24] [#] Walking from (40.778253, -73.97905) to (40.776883, -73.979461) for approx. 7.00 seconds
[2016-07-24 11:11:40] [#] Finished walking
[2016-07-24 11:11:41] [#] Arrived at Pokestop
[2016-07-24 11:11:44] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:11:57] [#] Found fort c2fb3819bbcc4a7db4810c71e49961c1.16 at distance 1.09km
[2016-07-24 11:11:57] [#] Need to move closer to Pokestop
[2016-07-24 11:11:57] [#] Walking from (40.776883, -73.979461) to (40.778533, -73.979608) for approx. 8.00 seconds
[2016-07-24 11:12:17] [#] Finished walking
[2016-07-24 11:12:18] [#] Arrived at Pokestop
[2016-07-24 11:12:21] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:12:35] [#] Found fort 22e3180ef66d43909bb0773efa4305b5.16 at distance 1.12km
[2016-07-24 11:12:35] [#] Need to move closer to Pokestop
[2016-07-24 11:12:35] [#] Walking from (40.778533, -73.979608) to (40.778054, -73.979972) for approx. 3.00 seconds
[2016-07-24 11:12:40] [#] Finished walking
[2016-07-24 11:12:41] [#] Arrived at Pokestop
[2016-07-24 11:12:45] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 11:13:04] [#] Scanning area for objects (3 / 25)
[2016-07-24 11:13:04] [#] Walking from (40.778054, -73.979972) to (40.7803766, -73.9641879) for approx. 55.00 seconds
[2016-07-24 11:15:33] [#] Finished walking
[#] (40.7803766, -73.9641879, 0)
[2016-07-24 11:15:42] [#] Scanning area for objects (4 / 25)
[2016-07-24 11:15:42] [#] Walking from (40.7803766, -73.9641879) to (40.7778766, -73.9641879) for approx. 12.00 seconds
[2016-07-24 11:15:49] [x] Exiting PokemonGo Bot

**Comment by MikeDX** [comment]
I believe this is because the login has expired (possibly wrong).

**Comment by ethervoid** [comment]
Related #643

**Comment by zx900930** [comment]
A stupid way to make it continue:
write a batch file or shell script to restart it every 60 mins:

example:
 `startup.bat`

```
@echo off
:loop
python pokecli.py
timeout /t 3600 >null
Taskkill /F /IM python.exe >nul
goto loop
```

**Comment by slonce** [comment]
Really with this script? I have run more than 10 python.exe-bot,map. This script kill they all...

**Comment by zx900930** [comment]
@slonce sorry for that i only running 1 python script, so the best way is let dev fix this issue in the bot.

**Comment by MikeDX** [comment]
That's a terrible "fix". I will look into a proper solution

**Comment by JohanJu** [comment]
Maybe its a scheduler issue, try set the priority of python.exe to high

**Comment by unequaled86** [comment]
#701 its fixed maybe

**Comment by surfaace** [comment]
duplicate of #643, maybe fixed in #701
